ansible/justfile

```diff
@@ -11,6 +11,27 @@ init env="staging":
 deploy env="staging":
   ansible-playbook -l {{ env }} deploy_site_playbook.yml
 
+# query the status of services for an environment
+status env="staging":
```

Comment:
nice, liking this just more and more :-)

Reply:
;)